prow/jobs/incubator/hack-showcase/slack-connector/slack-connector.yaml

```diff
@@ -0,0 +1,51 @@
+base_job_template: &base_job_template
+  skip_report: false
+  run_if_changed: "^slack-connector"
+  decorate: true
+  path_alias: github.com/kyma-incubator/hack-showcase
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd #???
```

Comment:
Change to `eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd:go1.12` or `eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd:go1.11` depending on which version you use locally.

Reply:
Simply without date and commit, `go1.11` or `go1.12`